specutils/io/default_loaders/wcs_fits.py

```diff
@@ -356,7 +483,7 @@ def _read_non_linear_iraf_fits(file_obj, spectral_axis_unit=None, flux_unit=None
 
     Returns
     -------
-    Tuple of data to pass to SpectrumCollection() or Spectrum1D():
+    Tuple of data to pass to `SpectrumCollection`() or `Spectrum1D`():
```

Comment:
```suggestion
    Tuple of data to pass to `~specutils.SpectrumCollection` or `~specutils.Spectrum1D`:
```

Reply:
p.s. Does not really matter because this is a hidden function, so not rendered by API doc. But might as well do it right?

Reply:
Yes (none of the default_loaders are rendered anywhere afaics).